addons/event/report/event_event_templates.xml

```diff
@@ -26,6 +26,8 @@
                                             <h3 t-field="o.name"/>
                                         </small>
                                     </div>
+                                     <div class="col-xs-12 text-center" style="background: lightgrey;" id="o_event_barcode">
```

Comment:
@msh-odoo 
we need to add ticket_type(event_Sale) and bar-code(event_barcode) in same grey light box and our module is just depends on event not event_Sale and event_Sale is also not depends on event_barcode so to add ticket type and  barcode in same box i have add one div in that.